CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.13.1 (2020-04-23)
+
+- Fixed the toggle switch styles to position native inputs correctly within `overflow` containers. [#114](https://github.com/blackbaud/skyux-theme/pull/114)
```

Comment:
Is `toggle switch styles` correct? The PR we link to shows changes to `sky-switch-input`, but it also links to an that is all about radio buttons, not toggle switches. Just making sure I'm not getting my wires crossed here.

Also, it's minor, but I'd move `correctly` right before `position`. I think this reads more naturally that way, but no big deal if you disagree.

Reply:
Done!